nav2_smac_planner/src/node_2d.cpp

```diff
@@ -154,11 +154,12 @@ bool Node2D::backtracePath(CoordinateVector & path)
 
   NodePtr current_node = this;
 
-  while (current_node->parent) {
+  do {
     path.push_back(
       Node2D::getCoords(current_node->getIndex()));
     current_node = current_node->parent;
   }
+  while (current_node->parent);
```

Comment:
Will have to wait for CI, but I think the while should be on the same line as the end bracket by styling, but LGTM if linting is happy!